### PR TITLE
Feature/register npm

### PR DESCRIPTION
**Pull request by youjung-hong**

### CODE REVIEW

**Comment by youjunghong** [comment]
ok to test

**Comment by youjunghong** [comment]
test

**Comment by youjunghong** [comment]
ok to test

**Comment by youjunghong** [comment]
ok to test

**Comment by youjunghong** [comment]
ok to test

**Comment by youjunghong** [comment]
ok to test

**Comment by youjunghong** [comment]
ok to test

**Comment by youjunghong** [comment]
ok to test

**Comment by youjung-hong** [comment]
ok to test

**Comment by youjung-hong** [comment]
ok to test

File: package.json [hunk 1]
```diff
@@ -1,56 +1,70 @@
...
+  "scripts": {
+    "test": "karma start --no-single-run",
+    "test:ne": "KARMA_SERVER=ne karma start",
+    "bundle": "webpack && webpack -p",
+    "serve": "webpack-dev-server --inline --hot -d",
+    "serve:ie8": "webpack-dev-server -d",
+    "cpy-dist2doc": "mkdir -p doc/dist && cp -f -r dist doc",
+    "doc": "jsdoc -c jsdoc.conf.json && npm run cpy-dist2doc"
+  },
   "description": "contextmenu component",
```
Comment (youjung-hong) [inline package.json:14]: TOAST UI Component: ContextMenu

File: package.json [hunk 2]
```diff
@@ -1,56 +1,70 @@
...
     "babel-plugin-transform-es2015-classes": "^6.7.7",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
     "babel-plugin-transform-es3-member-expression-literals": "^6.5.0",
     "babel-plugin-transform-es3-property-literals": "^6.5.0",
     "babel-preset-es2015": "^6.6.0",
-    "babelify": "^7.2.0",
-    "browserify": "^13.0.0",
-    "eslint": "^3.9.1",
+    "css-loader": "^0.26.1",
+    "es5-shim": "^4.5.9",
```
Comment (youjung-hong) [inline package.json]: 필요없음

File: package.json [hunk 3]
```diff
@@ -1,56 +1,70 @@
...
     "karma-es5-shim": "0.0.4",
     "karma-fixture": "^0.2.6",
     "karma-jasmine": "^1.0.2",
+    "karma-jasmine-jquery": "^0.1.1",
     "karma-junit-reporter": "^1.1.0",
-    "karma-webdriver-launcher": "git+https://github.com/nhnent/karma-webdriver-launcher.git",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webdriver-launcher": "git+https://github.com/nhnent/karma-webdriver-launcher.git#v1.1.0",
     "karma-webpack": "^1.8.0",
+    "safe-umd-webpack-plugin": "^0.0.1",
```
Comment (youjung-hong) [inline package.json]: 0.0.2 변경

File: package.json [hunk 4]
```diff
@@ -1,56 +1,70 @@
...
+    "jsdoc": "^3.5.4",
     "karma": "1.3.0",
-    "karma-browserstack-launcher": "^1.1.1",
     "karma-chrome-launcher": "^2.0.0",
-    "karma-coverage": "latest",
+    "karma-coverage": "^1.1.1",
     "karma-es5-shim": "0.0.4",
     "karma-fixture": "^0.2.6",
     "karma-jasmine": "^1.0.2",
+    "karma-jasmine-jquery": "^0.1.1",
```
Comment (youjung-hong) [inline package.json]: jquery, jasmine-jquery, karma-jasmine-jquery 이거 필요?

File: jsdoc.conf.json [hunk 5]
```diff
@@ -1,22 +1,26 @@
 {
     "source": {
-        "include": ["src", "README.md"],
+        "include": ["src/js", "README.md"],
```
Comment (youjung-hong) [inline jsdoc.conf.json]: js 빼기~

File: .eslintrc.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    "extends": "tui",
```
Comment (youjung-hong) [inline .eslintrc.js]: tui/es6

File: .eslintrc.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    "parserOptions": {
+        "sourceType": "module"
+    },
+    "env": {
+        "commonjs": true,
+        "jasmine": true
```
Comment (youjung-hong) [inline .eslintrc.js:8]: browser

File: .eslintrc.js [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    },
+    "globals": {
+        "util": true,
+        "dom": true,
```
Comment (youjung-hong) [inline .eslintrc.js]: util, dom 빼기

File: examples/example01-basic.html [hunk 9]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```
Comment (youjung-hong) [inline examples/example01-basic.html]: 파일명을 example01-basic-usage로!!!

File: README.md [hunk 10]
```diff
@@ -1,37 +1,73 @@
...
+* trigger a custom event, when click a menu item
+* make a menu item enable or disable
 
 ## Documentation
-* API: [https://nhnent.github.io/tui.component.contextmenu/latest](https://nhnent.github.io/tui.component.contextmenu/latest)
-* Tutorial: [https://github.com/nhnent/tui.component.contextmenu/wiki/ContextMenu-Tutorial](https://github.com/nhnent/tui.component.contextmenu/wiki/ContextMenu-Tutorial)
-* Examples: [https://nhnent.github.io/tui.component.contextmenu/latest/tutorial-default.html](https://nhnent.github.io/tui.component.contextmenu/latest/tutorial-default.html)
+* API: [https://nhnent.github.io/tui.context-menu/latest](https://nhnent.github.io/tui.context-menu/latest)
+* Tutorial: [https://github.com/nhnent/tui.context-menu/wiki](https://github.com/nhnent/tui.context-menu/wiki)
+* Examples: [https://nhnent.github.io/tui.context-menu/latest/tutorial-example01-basic.html](https://nhnent.github.io/tui.context-menu/latest/tutorial-example01-basic.html)
```
Comment (youjung-hong) [inline README.md]: example01-basic-usage

**Review by youjung-hong — COMMENTED** [review]
코드리뷰 끝

**Pull request merged**